.evergreen/run-kms-servers.sh

```diff
@@ -1,7 +1,7 @@
 cd ${DRIVERS_TOOLS}/.evergreen/csfle
 . ./activate_venv.sh
 # by default it always runs on port 5698
-./kmstlsvenv/bin/python3 -u kms_kmip_server.py &
-./kmstlsvenv/bin/python3 -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/expired.pem --port 8000 &
-./kmstlsvenv/bin/python3 -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/wrong-host.pem --port 8001 &
-./kmstlsvenv/bin/python3 -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/server.pem --port 8002 --require_client_cert &
+./kmstlsvenv/bin/python3 -u kms_kmip_server.py &> /dev/null &
+./kmstlsvenv/bin/python3 -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/expired.pem --port 8000  &> /dev/null &
+./kmstlsvenv/bin/python3 -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/wrong-host.pem --port 8001  &> /dev/null &
+./kmstlsvenv/bin/python3 -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/server.pem --port 8002 --require_client_cert &> /dev/null &
```

Comment:
I assume this is from noisey logs but could this end up hiding something important?

Reply:
I've never seen any important output, it more often than not just spits out random logs inside test output.  I'd rather suppress it for the normal use case and we can always remove it later if we need to.